### DISCUSSION TITLE
Why the ChipGroup does not has a ref property like CheckboxGroup does?

### DISCUSSION THREAD

**Discussion by AnsonWooBizCloud** [mantinedev/mantine#7262] [Q&A] [answered]
Hello everyone, I am new to Mantine UI, not sure if I should ask here, please let me know if I should ask somewhere else.

I thought it was like ChecboxGroup? I was going to use it with forwardRef but I cannot bind the ref because of the error below:
`Type '{ ref: any; disabled?: boolean | undefined; multiple?: false | undefined; children?: ReactNode; onBlur: Noop; name: Path<T>; value: PathValue<T, Path<T>>; onChange: (e: string) => void; }' is not assignable to type 'IntrinsicAttributes & ChipGroupProps<false>'.
  Property 'ref' does not exist on type 'IntrinsicAttributes & ChipGroupProps<false>'.ts(2322)`

Is it intended? Or it is a bug? Please advise, Thank You!!

**Comment by rtivital** [comment, marked as answer]
ChipGroup does not render any html elements, there is nothing to get ref off. If you want to get ref of an element that wraps all chips, add it on your side.
